client/src/routes/screens/LiveSession/ChangingRoom.tsx

```diff
@@ -191,12 +191,19 @@ const ChangingRoom = () => {
     session,
   ]);
 
-  const joinPress = useCallback(() => {
+  const joinPress = useCallback(async () => {
+    joinSession(session.inviteCode);
```

Comment:
Ah interesting approach. In the discussion we talked about doing this in the `preJoin` https://github.com/29ki/29k/blob/main/client/src/routes/screens/LiveSession/ChangingRoom.tsx#L149

But I guess this is better because it's when the user actually is committed. But FYI for all of us @swemail